spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -351,6 +351,44 @@ For example:
 ```
 stream create task-stream --definition "http --port=9000 | tasklaunchrequest-transform --uri=maven://org.springframework.cloud.task.app:timestamp-task:jar:1.2.0.RELEASE | task-launcher-local --maven.remote-repositories.repo1.url=http://repo.spring.io/libs-release"
 ```
+=== Launching a Composed Task From a Stream
+A composed task can be launched using one of the `task-launcher` sinks as discussed
+<<spring-cloud-dataflow-launch-tasks-from-stream, here>>.  Since we will be using
+the `ComposedTaskRunner` directly we will need to setup the task definitions it
+will use prior to the creation of the composed task launching stream.  So let's
+say that we wanted to create the following composed task definition `AAA && BBB`.
+The first step would be to create the task definitions.  For example:
+```
+task create CTR-AAA --definition "AAA"
```

Comment:
Why can't we simply create regular task definitions and combine the definitions in the composed-task graph DSL?

```
task create AAA --definition "timestamp"
task create BBB --definition "timestamp"
task create CTR --definition "AAA && BBB"
```

This would follow the general process of a composed-task orchestration in SCDF.

Reply:
If I understand your question correctly, DataFlow Launch command looks at the definition and determines whether the definition is a regular task or a CTR.  If it determines that it is a CTR it then builds up the graph based on the definition above (adding the prefixes) and establishes the app to use as CTR.  When using a stream the TLR-Transformer doesn't have the ability to determine if the definition is a CTR or not, so it just passes the definition as is to the TaskLaunchRequest.  So this is the reason, we used the definition as below.

Reply:
I will update the sample to use timestamp instead of just `AAA`  & `BBB` as the apps for the definitions.